Tools/GenerateWebsiteData.ps1

```diff
@@ -253,22 +263,31 @@ function Collapse-ParameterDescriptions($parameters) {
     }
 }
 
+# Generate a ProductInfo object. The $isBeta flag should only be used if every cmdlet in the
+# GCP product is in Beta. Otherwise, we'll just mark individual cmdlets as beta or not.
+# TODO(chrsmith): We can do this analysis after the fact (using the $isBeta field on cmdlets)
+# so we can obviate hard-coding this here.
+function GenerateProductInfo($longName, $shortName, $isBeta = $false) {
+    return @{
+        name = $longName;
+        shortName = $shortName;
+        isBeta = $isBeta;
+        resources = @()
+    }
+}
+
 # Mapping of cmdlet prefixes to Google Cloud Platform productInfo objects.
 $productInfoLookup = @{
-    "Gcs"   = @{ name = "Google Cloud Storage";  shortName = "google-cloud-storage" ; resources = @() }
-    "Gce"   = @{ name = "Google Compute Engine"; shortName = "google-compute-engine"; resources = @() }
-    "GcSql" = @{ name = "Google Cloud SQL";      shortName = "google-cloud-sql"     ; resources = @() }
-    "Gcd"   = @{ name = "Google Cloud DNS";      shortName = "google-cloud-dns"     ; resources = @() }
-    "Gcps"  = @{ name = "Google Cloud PubSub";      shortName = "google-cloud-pubsub"     ; resources = @() }
-    "GcLog" = @{ name = "Google Cloud Logging";      shortName = "google-cloud-logging"     ; resources = @() }
-    "GcIam" = @{ name = "Google Cloud IAM";
-                   shortName = "google-cloud-iam";
-                   resources = @() }
-    "GcpProject"   = @{ name = "Google Cloud Project";
-                   shortName = "google-cloud-project";
-                   resources = @() }
-    "Gke"   = @{ name = "Google Container Engine";  shortName = "google-cloud-container" ; resources = @() }
-    "Bq"    = @{ name = "Google Cloud BigQuery";  shortName = "google-cloud-bigquery" ; resources = @() }
+    "Gcs"   = GenerateProductInfo      "Google Cloud Storage"    "google-cloud-storage"
+    "Gce"   = GenerateProductInfo      "Google Compute Engine"   "google-compute-engine"
+    "GcSql" = GenerateProductInfo      "Google Cloud SQL"        "google-cloud-sql"
+    "Gcd"   = GenerateProductInfo      "Google Cloud DNS"        "google-cloud-dns"
+    "Gcps"  = GenerateProductInfo      "Google Cloud PubSub"     "google-cloud-pubsub"
+    "GcLog" = GenerateProductInfo      "Google Cloud Logging"    "google-cloud-logging"
+    "GcIam" = GenerateProductInfo      "Google Cloud IAM"        "google-cloud-iam"
+    "GcpProject" = GenerateProductInfo "Google Cloud Project"    "google-cloud-project"
+    "Gke"   = GenerateProductInfo      "Google Container Engine" "google-cloud-container"    $true
+    "Bq"    = GenerateProductInfo      "Google Cloud BigQuery"   "google-cloud-bigquery"
```

Comment:
Should we add $true to BigQuery, Logging, GcIam, GcpProject and Pubsub too since they are still in Beta?